README.md

```diff
@@ -23,11 +23,86 @@ Your new file structure should look like:
     - Hermes
       - src
 
+## RabbitMQ Broker
+
+### Getting setup
+
+1) Installing rabbitMQ
+
+In order to send messages, you must have a rabbitMQ server to send the messages to.
+This can be installed [here](https://www.rabbitmq.com/)
```

Comment:
Most package managers should have rabbit-mq. You should indicate the `apt-get` way to install it as well.